### PR TITLE
Feature/add docs for cookies

### LINKED ISSUE #660: Documentation for a new CookieJar's methods
# What?

There is a PR https://github.com/grafana/k6/pull/2509 that introduces new methods to a `CookieJar`. To keep our docs up to date, we need to document them once PR will be close to merging.

# Why?

It's important to maintain the [documentation up to date](https://k6.io/docs/javascript-api/k6-http/cookiejar/).

### PR DESCRIPTION
**Pull request by Maksimall89**
resolved #660

### CODE REVIEW

**Comment by Maksimall89** [comment]
Hey, @ppcano can you check my PR?

**Comment by ppcano** [comment]
@Maksimall89, thanks also for updating the docs for this new APIs. 

The docs use a linter to validate the k6 examples and be consistent with the JS convention. The linting error is in one of the examples.  

Please, run`npm run lint:examples:fix` to fix the issue and commit the changes.

**Comment by ppcano** [comment]
Note that this PR will be merged after https://github.com/grafana/k6/pull/2509 is included on a k6 release.

**Comment by Maksimall89** [comment]
@ppcano thanks for the information. I fixed the issue.

**Review by olegbespalov — APPROVED** [review]
LGTM :+1:

**Comment by mstoykov** [comment]
Note: this needs to be merged *before* we release the v0.39.0 branch, but not before that.

**Review by codebien — APPROVED** [review]

File: src/data/markdown/docs/05 Examples/01 Examples/08 cookies-example.md [hunk 1]
```diff
@@ -115,8 +115,82 @@ export default function () {
...
+  });
+}
+```
+
+</CodeGroup>
+
+## Clear all cookies in the VU cookie jar
+
+To clear all cookies in the jar by specifying url
+you'd do something like this:
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/05 Examples/01 Examples/08 cookies-example.md:160]:
```suggestion
To clear all cookies in the jar by specifying url, use the `clear` method.
```

File: src/data/markdown/docs/05 Examples/01 Examples/08 cookies-example.md [hunk 2]
```diff
@@ -115,8 +115,82 @@ export default function () {
 
 </CodeGroup>
 
+## Delete a cookie in the VU cookie jar
+
+To delete a cookie in the jar by specifying url and name
+you'd do something like this:
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/05 Examples/01 Examples/08 cookies-example.md:121]:
```suggestion
To delete a cookie in the jar for a specific URL and name, use the `delete` method.
```

**Review by MattDodsonEnglish — COMMENTED** [review]
@mstoykov 

Can we merge this now? We said to release it after 0.39.0 and we're almost at 0.40.0.

These docs look good to me!
For the merge conflict, though, I'd recommend just using the description that already exists.

**Comment by MattDodsonEnglish** [comment]
@Maksimall89 I forgot, can you also sign the CLA! That's a new policy for contributors to all Grafana repos.

**Comment by MattDodsonEnglish** [comment]
@Maksimall89 Thanks for signing the CLA, and especially thanks for this great work on the docs! :muscle:

**Pull request merged**